src/account/__tests__/accountsReducer-test.js

```diff
@@ -153,67 +72,74 @@ describe('accountsReducer', () => {
   });
 
   describe('LOGIN_SUCCESS', () => {
-    const account1 = eg.makeAccount({ email: '', realm: new URL('https://one.example.org') });
-    const account2 = eg.makeAccount({ realm: new URL('https://two.example.org') });
+    const account1 = eg.makeAccount({
+      realm: new URL('https://one.example.org'),
+      ackedPushToken: eg.randString(),
+    });
+    const account2 = eg.makeAccount({
+      realm: new URL('https://two.example.org'),
+      ackedPushToken: eg.randString(),
+    });
 
     const prevState = deepFreeze([account1, account2]);
 
-    test('on login, update initial account with auth information, without clobbering zulipVersion', () => {
-      const newAccount = eg.makeAccount({
-        realm: account1.realm,
-        zulipVersion: null,
-      });
+    test("on login, update initial account with apiKey, set ackedPushToken to null, don't clobber anything else", () => {
```

Comment:
I think this test can be dropped now as a duplicate of "on login, if account does exist, …". By "initial account" I think the description means the thing this PR gets rid of -- the storage of partial login information here in Redux.